bugsnag-plugin-android-ndk/src/main/jni/bugsnag_ndk.c

```diff
@@ -42,6 +42,26 @@ bsg_unwinder bsg_configured_unwind_style() {
   return BSG_CUSTOM_UNWIND;
 }
 
+void bugsnag_add_on_error_env(JNIEnv *env, bsg_on_error on_error) {
+  if (bsg_global_env != NULL) {
+    bsg_global_env->on_error = on_error;
+  }
+}
+
+void bugsnag_remove_on_error_env(JNIEnv *env, bsg_on_error on_error) {
+  if (bsg_global_env != NULL) {
+    bsg_global_env->on_error = NULL;
+  }
+}
+
+bool bsg_run_on_error(bsg_environment *env) {
+  bsg_on_error on_error = env->on_error;
```

Comment:
Other methods in this file use `bsg_global_env` directly and all callers pass this in as the value, so I would suggest we keep the established pattern and remove the parameter.